prepare.sh

```diff
@@ -49,6 +49,7 @@ while true; do
         ;;
     --)
         shift
+        exit 0
```

Comment:
is this hit when parameters are passed that are not recognized? If so, please display an error and `exit 1`